aviary/variable_info/variable_meta_data.py

```
@@ -2518,7 +2518,7 @@
     historical_name={"GASP": 'INPROP.DPROP', "FLOPS": None, "LEAPS1": None},
     units='ft',
     desc='propeller diameter',
-    default_value=0.0,
+    default_value=8.0,
```

Do we really want a non-zero default value for this? It could mask a problem where when we are missing a promote (this happened recently with MBSAE).

---

I'm not sure if we have a good answer to this, we've already been moving defaults out of individual component add_inputs/add_outputs to the metadata to centralize everything, which does conflict with your concerns. At the very least, this is becoming consistent with the current version of how we're handling defaults.

---

I prefer keep `default_value=0.0`.